### DISCUSSION TITLE
Need help training with fine-tuning (pls)

### DISCUSSION THREAD

**Discussion by voltioDS3** [tloen/alpaca-lora#79] [Q&A]
Long story short: my friend and I have fine-tuned with a custom data.json till checkpoint-200  but now we don't know how to run it. We dont really understand how use the model we trained to input prompts and get answers, we just know that after running finetune.py a folder was created with the name "lora-alpaca", inside it has a "checkpoint-200" folder, a "run" folder, an "adapter_config.json" and an "adapter_model.bin". Please we would be really thankful if someone could help us. 

 
![image](https://user-images.githubusercontent.com/86634774/226225217-a6a3c588-2fce-48fb-8bf5-4dda02837ba4.png)

**Comment by AngainorDev** [comment]
The final model is the two adapter_*.* files.

Use generate.py to run your lora.
Change the lora path from `LORA_WEIGHTS = "tloen/alpaca-lora-7b"` to `LORA_WEIGHTS = "./lora-alpaca"` , you local lora.

**Reply by OfficerChul** [reply]
I think you should also specify the checkpoint path for example `LORA_WEIGHTS = "./lora-alpaca/checkpoint-200"`
